lib/amorail/entity/params.rb

```diff
@@ -17,11 +17,10 @@ def custom_fields
       props = properties.send(self.class.amo_name)
 
       custom_fields = []
-
-      self.class.properties.each do |k, v|
+      self.class.properties.each do |k, value|
         prop_id = props.send(k).id
-        prop_val = { value: send(k) }.merge(v)
-        custom_fields << { id: prop_id, values: [prop_val] }
+        prop_val = props.send(k)['type_id'] == "5" ? [send(k)] : { value: send(k) }.merge(value)
```

Comment:
I changed it to use a constant instead. It's the field id for a multiselect type of field.
https://developers.amocrm.com/rest_api/accounts_current.php#type_id